querybook/webapp/components/DataDocCell/DataDocCell.tsx

```diff
@@ -80,6 +92,60 @@ export const DataDocCell: React.FunctionComponent<IDataDocCellProps> =
                 fullScreenCellIndex,
             } = useContext(DataDocContext);
 
+            let queryCells = useQueryCells(docId);
+            const queryTitle = queryCells[index].meta.title;
```

Comment:
im not sure if you can just use the index here? if you have 2 text cells above this, then this index is +2 of the actual query cell index

Reply:
use queryIndexInDoc